mmcv/cnn/utils/weight_init.py

```diff
@@ -45,14 +46,17 @@ def update_init_info(module, init_info):
             module._params_init_info[param]['tmp_mean_value'] = mean_value
 
 
-def constant_init(module, val, bias=0):
+def constant_init(module: nn.Module, val: float, bias: float = 0) -> None:
```

Comment:
Maybe Union[int, float?]. The type of default value is int

Reply:
Fixed